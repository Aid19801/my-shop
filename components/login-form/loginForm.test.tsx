import { render, screen } from "@testing-library/react";
import { LoginForm } from "./";

describe("LoginForm", () => {
  it("should render the component without exploding", () => {
    render(<LoginForm />);
    const form = screen.getByTestId("login-form");
    expect(form).toBeInTheDocument();
  });
  it("should match snapshot", () => {
    const { container } = render(<LoginForm />);
    expect(container).toMatchSnapshot();
  });
  it("should capture the user's username as expected", () => {
    render(<LoginForm />);
    expect(container).toMatchSnapshot();
  });
});
