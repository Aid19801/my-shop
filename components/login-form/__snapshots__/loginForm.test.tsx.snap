// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm should match snapshot 1`] = `
<div>
  <form
    class="form"
    data-testid="login-form"
  >
    <input
      name="username"
      placeholder="username"
      type="text"
    />
    <input
      name="password"
      type="password"
    />
    <button
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
